specification/network/resource-manager/Microsoft.Network/stable/2019-07-01/examples/IpTagsUpdate.json

```diff
@@ -0,0 +1,51 @@
+{
+    "parameters": {
+      "ipTagName": "ipTags1",
+      "resourceGroupName": "rg1",
+      "api-version": "2019-07-01",
+      "subscriptionId": "subid",
+      "ipTagsParameters": {
+        "tags": {
```

Comment:
This is an example for updating tags, but in response those new tags aren't present

Reply:
Done